### PR TITLE
Building an animated vertical timeline chart with HTML, CSS and JavaScript.

### LINKED ISSUE #6756: BuIldIng an animated Vertical Timeline Chart with HTML, CSS (Glassmorphism), and JavaScript.
*NOTE*: All **1st-time contributors** should know Topic approval and PR submission does NOT guarantee your Topic/Article will be published. Our team of Peer Reviewers and Content Moderators will review all PRs that come in to make sure they adhere to the standard of quality we expect from the EngEd community. 

We expect all community memebers to go through provided examples, resources, and previously published material before submitting content. As a rule of thumb, please only submit articles (pull requests) that are complete, formatted correctly, and include a fully polished article (ready to be published and error free). 

All revisions and edits should be *completed on your own forked repo (as to not take up room in the queue)*. **Any PR that is submitted incomplete (meaning not ready to be published as is) will be closed.** A PR being closed does NOT mean the article can not be published (or fixed) it simply means that the edits, revisions, and fixes will happen outside of the queue.

**NOTE: Any topic suggestion copied/taken/modified from YouTube may Lead to a permanent ban from the community.**

### Topic Suggestion
Enter topic suggestion here...

#### Pre-submission advice
By following all our pre-submission advice and reviewing our Resources folder, you will maximise your chances of your topic being approved.

We ask that you please be patient as our team works through approving and publishing all articles/tutorials in a timely manner.

Allow 1-3 days for a topic to be reviewed and/or approved - allow 3-7 days for an articles to be reviewed and/or published (subject to vary depending on _volume and/or backlog of articles_).

Be sure to visit our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for tools, resources, and example articles that will help you propose and write a successful article.

**Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**We tend to stray away or tend not to publish reviews/comparisons of commercial product offerings.**

### Writing sample(s): 
Include any links or writing samples - to help our team better gauge your writing quality.

#### Proposed title of article
BuIldIng an animated Vertical Timeline Chart with HTML, CSS (Glassmorphism), and JavaScript.

Use keyword research to improve your article's SEO.

#### Proposed article introduction

Have you ever considered making a visual representation of your accomplishments throughout your career? If that's the case, you'll only need a timeline chart. You may easily incorporate all of your accomplishments in a single link using a timeline. The résumé, portfolio, and timeline chart are the primary visual representations of our accomplishments and experiences.

In this article, we'll utilize *HTML5, CSS3 (glassmorphism implementation), and Vanilla JavaScript* to create a vertical timeline chart for storing our accomplishments and experiences.

For example, if the article is based on machine learning, use the following: [Machine learning] Introduction to Machine Learning.
If the article is based on developing an Android application, use the following: [Android] Developing Apps using Android.

ONLY viable topics to pick from:
- API
- Artificial Intelligence
- Blockchain
- Containers
- Edge Computing
- Languages
- Machine Learning
- Networking
- Node.js
- Security

#### Key takeaways 
1.) The reader will learn how to implement glassmorphism
2.) The reader will learn how to use css3 couple with JavaScript to perform the animation
3.) Readers will learn how to make a function active in JavaScript without invoking/calling it
4.) Readers will also learn how to deploy their timeline chart using github.

#### Article quality
In this article, I'll walk you through the process of creating this web app step by step. What distinguishes my article from others is that I will explain how to add images to your timeline (for example, an image taken at an event you attended) and how to embed links (for example, an article you wrote or a link to a GitHub repo you once contributed to), both of which are topics that other articles do not cover.

#### References
Please list links to any published content/research that you intend to use to support/guide this article.

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by shegz101**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #6756

### CODE REVIEW

**Comment by shegz101** [comment]
@BrianaNzivu please has anyone started the plagiarism check on this content, so you can proceed with the review. I just hope this reviewing process will be fast.

**Comment by BrianaNzivu** [comment]
> @BrianaNzivu please has anyone started the plagiarism check on this content, so you can proceed with the review. I just hope this reviewing process will be fast.

@section-engineering-education/content-moderator-team .

Hey @shegz101. A check will be done ASAP.

Regards.

**Comment by louisefindlay23** [comment]
Plagiarism check complete. Please fix the extra line breaks in between your HTML code snippets.

**Comment by hectorkambow** [comment]
@BrianaNzivu Lets be sure to glance at the content before we dive into our review to see if any glaring issue exist.

File: content/articles/building-an-animated-vertical-timeline-chart-with-html-css-and-javascript/index.md [hunk 1]
```diff
@@ -0,0 +1,936 @@
...
+</main>
+…
+
+```
+
+Within the `main` tag, create another `section` tag with a class named `timeline-section` that will contain the events for your timeline.
+
+```HTML
+
+…
```
Comment (BrianaNzivu) [inline content/articles/building-an-animated-vertical-timeline-chart-with-html-css-and-javascript/index.md:88]: Indent your code well.

Comment (BrianaNzivu) [inline content/articles/building-an-animated-vertical-timeline-chart-with-html-css-and-javascript/index.md:88]: Remove this...for all code snippets

**Review by BrianaNzivu — CHANGES_REQUESTED** [review]
Hey @shegz101 👋 
Good job on the article.
Just a few things I would love to point out before we proceed with the preliminary revisions:
- Your article has quite a lot of mistakes. I suggest you use [Grammarly](https://app.grammarly.com/) to make these corrections. 
- Please structure your article as required by Section. You can check out a tutorial's structures [here](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md).
- Refer to the comments made below.


Ping me when done.

Kind Regards.

**Comment by BrianaNzivu** [comment]
@section-engineering-education/peer-reviewers 👋 
I am noticing this article is using Vanilla Js which I am not proficient in. 

Someone who is good at it please feel free to take it up.

**Comment by shegz101** [comment]
@BrianaNzivu I have made all the corrections. And I have already linked this PR to an issue initially.
Thanks.

**Comment by shegz101** [comment]
@BrianaNzivu please are you still reviewing the article, I have done all you said I should do.

**Comment by shegz101** [comment]
> @section-engineering-education/peer-reviewers 👋 I need someone to help review this article, its' been 11 days since I opened this PR.

**Comment by louisefindlay23** [comment]
@shegz101, have correctly linked your PR to your topic suggestion for you. Please remember in the future to follow the instructions in the Article Checklist under Topic Suggestion This Closes.

**Comment by shegz101** [comment]
> @shegz101, have correctly linked your PR to your topic suggestion for you. Please remember in the future to follow the instructions in the Article Checklist under Topic Suggestion This Closes.

Thank you, I will properly link it next time. Don't know if you can help review this article.

**Comment by BrianaNzivu** [comment]
Hey @shegz101 👋
Since no one has taken up the article. I will review it but I am requesting if we can work closely on Slack for any clarifications and questions that I might have especially on the Java script end.

Please DM me ASAP. 
Regards.

**Comment by shegz101** [comment]
@BrianaNzivu  please what's your name on slack.

**Comment by shegz101** [comment]
@BrianaNzivu please what's your name on slack.

**Comment by BrianaNzivu** [comment]
Briana Nzivu

**Comment by shegz101** [comment]
OK let me try again, because I wasn't getting your name before.

**Comment by shegz101** [comment]
@BrianaNzivu just sent you a message on slack

**Comment by BrianaNzivu** [comment]
Proceeding to tech review 👍 

@shegz101 please check your DM.

**Comment by shegz101** [comment]
@BrianaNzivu please what's the latest update concerning the tech review.

**Comment by BrianaNzivu** [comment]
Tech review is done and code functioning well 👍

**Comment by shegz101** [comment]
> Tech review is done and code functioning well 👍

Thanks, please what's next.

**Review by BrianaNzivu — APPROVED** [review]
Hey @shegz101👋 .

Very good job on the article. It is a very interesting read👏 .

Just a few things I'd like to point out:
- I suggest that you check out the [structure](https://github.com/section-engineering-education/engineering-education/tree/master/new_contributors) for the various articles accepted by Section.
- You check out our resources page which has links to various resourceful tools for writing just to minimize the changes being made by the reviewer.
- Lastly please go through ALL the corrections I have made so that you can not repeat the same mistakes.

That's it! :smile: 

A final review will be done now by a content moderator.
Otherwise, congratulations on publishing your article🍾.

In case of any question or clarification (currently or in the future), do not hesitate to reach out via [Slack](https://app.slack.com/client/T5X14SH7G/D01A6MT673R) or my personal email (briananzivu@gmail.com). 
I am **ALWAYS** here to help.

It has been a pleasure reading and reviewing this article.

Looking forward to seeing you contribute more to EngEd. 🚀

Once again good job! 👍 

Kind Regards

**Comment by shegz101** [comment]
> Hey @shegz101👋 .
> 
> Very good job on the article. It is a very interesting read👏 .
> 
> Just a few things I'd like to point out:
> 
> * I suggest that you check out the [structure](https://github.com/section-engineering-education/engineering-education/tree/master/new_contributors) for the various articles accepted by Section.
> * You check out our resources page which has links to various resourceful tools for writing just to minimize the changes being made by the reviewer.
> * Lastly please go through ALL the corrections I have made so that you can not repeat the same mistakes.
> 
> That's it! 😄
> 
> A final review will be done now by a content moderator. Otherwise, congratulations on publishing your article🍾.
> 
> In case of any question or clarification (currently or in the future), do not hesitate to reach out via [Slack](https://app.slack.com/client/T5X14SH7G/D01A6MT673R) or my personal email ([briananzivu@gmail.com](mailto:briananzivu@gmail.com)). I am **ALWAYS** here to help.
> 
> It has been a pleasure reading and reviewing this article.
> 
> Looking forward to seeing you contribute more to EngEd. 🚀
> 
> Once again good job! 👍
> 
> Kind Regards

Thanks Briana I would definitely reach out.

**Comment by hectorkambow** [comment]
> Hey @shegz101 👋 Good job on the article. Just a few things I would love to point out before we proceed with the preliminary revisions:
> 
> * Your article has quite a lot of mistakes. I suggest you use [Grammarly](https://app.grammarly.com/) to make these corrections.
> * Please structure your article as required by Section. You can check out a tutorial's structures [here](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md).
> * Refer to the comments made below.
> * Please link the issue this PR fixes.
> 
> Ping me when done.
> 
> Kind Regards.

@BrianaNzivu 

The article should have been closed at this stage 💯

**Comment by hectorkambow** [comment]
jumping on this one for final review

**Comment by hectorkambow** [comment]
Was not able to preview (render on local host)

**Comment by hectorkambow** [comment]
@section-engineering-education/content-moderator-team Is anyone able to test to see if they can preview on their end? 
Final review was complete

**Comment by shegz101** [comment]
@srishilesh please can you help with this article, it is not rendering.

**Comment by BrianaNzivu** [comment]
> @section-engineering-education/content-moderator-team Is anyone able to test to see if they can preview on their end? 
> Final review was complete

Cc @section-engineering-education/content-moderator-team

**Comment by shegz101** [comment]
@BrianaNzivu please try to render and let me see the error. Thanks

**Comment by shegz101** [comment]
@srishilesh please help me check out the issue with this PR, it's not rendering on local host (Hugo).

**Comment by srishilesh** [comment]
> @srishilesh please help me check out the issue with this PR, it's not rendering on local host (Hugo).

Hey @shegz101 ,

Thanks for reaching out.

I have been having some issues with my laptop for the past 2 days. I'll get to this issue in a day or two.

Meanwhile, I suggest you to take a look [at this guide](https://github.com/section-engineering-education/engineering-education/blob/master/content-moderators/how-to-preview-hugo.md) and try it out in your system. Let me know if some error comes up.

Thanks.

**Comment by BrianaNzivu** [comment]
> @BrianaNzivu please try to render and let me see the error. Thanks

Hey @shegz101 👋🏾.
You can try creating a new PR on your latest forked repo kmake sure it's up to date with our master) . It might be merge issues.
I will close this to allow you to open another PR. Once you open the PR please ping me or request a review for me so that we see whether we will fix this issue.

Regards.
Cc @hectorkambow
@srishilesh
